src/commands/asset-compute/devtool.js

```diff
@@ -12,96 +12,23 @@
 
 'use strict';
 
+const devtool = require('@adobe/asset-compute-devtool');
+
 const BaseCommand = require('../../base-command');
 const { flags } = require('@oclif/command');
-const app = require('@adobe/asset-compute-devtool/app');
-const http = require('http');
-const { createHttpTerminator } = require('http-terminator');
-const util = require('../../lib/util');
-const open = require('open');
-const getPort = require('get-port');
-const crypto = require("crypto");
 
-/**
- * Event listener for HTTP server "listening" event.
- */
-async function onListening(server, randomString) {
-    const addr = server.address();
-    const bind = typeof addr === 'string'
-        ? 'pipe ' + addr
-        : 'port ' + addr.port;
-    util.log('Listening on ' + bind);
-    const assetComputeDevToolUrl = `http://localhost:${addr.port}/?devToolToken=${randomString}`;
-    console.log('Asset Compute Developer Tool Server started on url', assetComputeDevToolUrl);
-    await open(assetComputeDevToolUrl);
-}
 
-/**
- * Normalize a port into a number, string, or false.
- */
-async function findOpenPort(preferredPort) {
-    return getPort({port: [preferredPort, preferredPort + 1, preferredPort + 2]});
-    // Will use specified port if available, otherwise fall back to a random port
-}
+require('dotenv').config;
```

Comment:
this is actually not needed since it is loaded in the `index.js` in the devtool: https://github.com/adobe/asset-compute-devtool/blob/master/server/index.js#L17